guides/common/modules/proc_renaming-smart-proxy.adoc

```diff
@@ -72,7 +73,8 @@ For more information, see {ManagingHostsDocURL}Configuring_and_Setting_Up_Remote
 # subscription-manager refresh
 ----
 +
-. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
-. Locate {SmartProxyServer} in the list, and click *Edit* to the right of it.
-. Edit the *Name* and *URL* fields to match {SmartProxyServer}'s new host name, then click *Submit*.
-. On your DNS server, add a record for {SmartProxyServer}'s new host name, and delete the record for the previous host name.
+. Update the {SmartProxy} host name in the {ProjectWebUI}.
```

Comment:
We generally avoid nested lists. It'd be fine for me if there's no objection from Lena or Adam.

Maybe remove nesting and add "Perform this procedure for all {SmartProxyServers}" or similar?

Reply:
Is there a reason why you generally avoid nested lists? Just so I know.

I actually don't mind using them at all as long as they're only 2 levels deep like here. And in this case, I do think it makes sense to use it because the substeps are part of the web UI step, which is something the nesting makes clear at first glance.

Reply:
I am not sure myself where it comes from, but Melanie told me/us some time ago.

Reply:
Ok, thanks a lot! I'll check the IBM style guide and Red Hat internal style guide, I might also just ask around. If I don't find anything, I'll keep it as it is.